### PR TITLE
tests for woopra and segment

### PR DESCRIPTION
**Pull request by tinakurian**
Paired to: https://github.com/codeready-toolchain/registration-service/pull/150
Related to: https://github.com/codeready-toolchain/host-operator/pull/334
Jira:  https://issues.redhat.com/browse/CRT-853

This PR tests the two new endpoints; one for retrieving the woopra domain and another for retrieving the segment-write-key.

### CODE REVIEW

File: test/e2e/registration_service_test.go [hunk 1]
```diff
@@ -100,6 +100,58 @@ func (s *registrationServiceTestSuite) TestHealth() {
...
+
+		mp := make(map[string]interface{})
+		err = json.Unmarshal([]byte(body), &mp)
+		require.NoError(s.T(), err)
+
+		// Verify JSON response.
+		woopraDomain := mp["segment-write-key"]
+		require.Equal(s.T(), "test segment write key", woopraDomain)
+	})
+}
```
Comment (alexeykazakov) [inline test/e2e/registration_service_test.go:153]: Or something like:

```suggestion
func (s *registrationServiceTestSuite) TestWoopra() {
	assertNotSecuredGetResponseEquals := func(endPointPath, expectedResponseKey, expectedResponseValue string) {
		// Call woopra domain endpoint.
		req, err := http.NewRequest("GET", fmt.Sprintf("%s/api/v1/%s", s.route, endPointPath), nil)
		require.NoError(s.T(), err)

		resp, err := httpClient.Do(req)
		require.NoError(s.T(), err)
		defer close(s.T(), resp)

		assert.Equal(s.T(), http.StatusOK, resp.StatusCode)

		body, err := ioutil.ReadAll(resp.Body)
		require.NoError(s.T(), err)
		require.NotNil(s.T(), body)

		mp := make(map[string]interface{})
		err = json.Unmarshal([]byte(body), &mp)
		require.NoError(s.T(), err)

		// Verify JSON response.
		require.Equal(s.T(), expectedResponseValue, mp[expectedResponseKey])
	}
	
	s.Run("get woopra domain 200 OK", func() {
		// Call woopra domain endpoint.
		assertNotSecuredGetResponseEquals("woopra-domain", "segment-write-key", "test woopra domain")
	})

	s.Run("get segment write key 200 OK", func() {
		// Call segment write key endpoint.
		assertNotSecuredGetResponseEquals("segment-write-key", "woopra-domain", "test segment write key")
	})
}
```

**Review by alexeykazakov — COMMENTED** [review]
Looks good!

File: deploy/host-operator/host-operator-config-map.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: host-operator-config
+type: Opaque
+data:
+  woopra.domain: "test woopra domain"
+  segment.write.key: "test segment write key"
```
[the remark this answers is not in the document]
Comment (tinakurian) [reply deploy/host-operator/host-operator-config-map.yaml]: fixed here: 5e40f56af24edb54017774cd772c80f0e43770f3

**Review by alexeykazakov — APPROVED** [review]
Looks good!
You can consider applying https://github.com/codeready-toolchain/toolchain-e2e/pull/205#discussion_r533866985 though :

**Comment by tinakurian** [comment]
> Looks good!
> You can consider applying [#205 (comment)](https://github.com/codeready-toolchain/toolchain-e2e/pull/205#discussion_r533866985) though :)

Thanks @alexeykazakov maybe it didn't show up properly? i clicked commit, looks like it was committed: fa2ed03f37ea285325e7172c349520213137a091

**Comment by alexeykazakov** [comment]
> Thanks @alexeykazakov maybe it didn't show up properly? i clicked commit, looks like it was committed

It was. I just commented seconds before you committed it :)

File: host-operator-config-map.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+
```
Comment (alexeykazakov) [inline host-operator-config-map.yaml:1]:
```suggestion
```

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**